<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryManager</class>
 <widget class="QDialog" name="InventoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Management</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="AddItemsTab">
    <attribute name="title">
     <string>Add Items</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Item Name :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantity :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="itemNameField">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="descriptionField">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="priceField">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="quantityField">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Item has a 'deal' price</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="dealEditor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Deal Editor</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantity of item necissary for deal :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dealQuantity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Price of deal quantity :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dealPrice">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addItemButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>440</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
    <widget class="QLabel" name="userInfoLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>651</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>440</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Fields</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="EditItemsTab">
    <attribute name="title">
     <string>Edit Items</string>
    </attribute>
    <widget class="QListWidget" name="inventoryList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Item</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="itemInformationBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>461</width>
       <height>471</height>
      </rect>
     </property>
     <property name="title">
      <string>Item Information</string>
     </property>
     <widget class="QLineEdit" name="itemPrice">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="itemDescription">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock Quantity :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Price :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemQuantity">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>40</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Item Name :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="editDealCheckBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Item has deal price</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="editDealEditor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Deal Editor</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantity of item necissary for deal :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editPageDealQuantity">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Price of deal quantity :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editPageDealPrice">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="cancelEditButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>320</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveAddButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteItemButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Item</string>
      </property>
     </widget>
     <widget class="QLabel" name="editPageUserInfoLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="editCloseButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <zorder>itemPrice</zorder>
     <zorder>itemDescription</zorder>
     <zorder>itemName</zorder>
     <zorder>label_10</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>itemQuantity</zorder>
     <zorder>label_7</zorder>
     <zorder>editDealCheckBox</zorder>
     <zorder>editDealEditor</zorder>
     <zorder>label_12</zorder>
     <zorder>label_11</zorder>
     <zorder>dealQuantity_2</zorder>
     <zorder>dealPrice_2</zorder>
     <zorder>cancelEditButton</zorder>
     <zorder>saveAddButton</zorder>
     <zorder>deleteItemButton</zorder>
     <zorder>editPageUserInfoLabel</zorder>
     <zorder>editCloseButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
